File: DotNetCore/Lab3_Task1_Sln/Lab3_Task1_Pro/Models/Student.cs
namespace Lab3_Task1_Pro.Models
{
    public class Student
    {
        public int Id { get; set; }
        public String Name { get; set; }
        public int Age { get; set; }
        public String Department { get; set; }
        public float Cgpa { get; set; }

        public Student(int id, string name, int age, string department, float cgpa)
        {
            Id = id;
            Name = name;
            Age = age;
            Department = department;
            Cgpa = cgpa;
        }

    }
}

File: Java_OOP1/MidTerm Task 1/Student/Student.java
// ID: 23-54110-3
// MD. MINHAJ ROWFUN RABBI ANIK
public class Student {
    private String name;
    private String id;
    private String department;
    private float cgpa;
    
    // Constructor
    Student(){
        System.out.println("E-Constructor");
    }
    Student(String name, String id, String department, float cga){
        System.out.println("P-Constructor");
        setName(name);
        setId(id);
        setDepartment(department);
        setCgpa(cga);
    }

    // Setter Methods
    public void setName(String name){
        this.name=name;
    }
    public void setId(String id){
        this.id=id;
    }
    public void setDepartment(String department){
        this.department=department;
    }
    public void setCgpa(float cgpa){
        this.cgpa=cgpa;
    }

    // Getter Methods
    public void showInfo(){
        System.out.println("Name: "+name);
        System.out.println("ID: "+id);
        System.out.println("Department: "+department);
        System.out.println("CGPA: "+cgpa+"\n");
    }
}
